types/constants.ts

```diff
@@ -1 +1,62 @@
-export const ARNS_NAME_REGEX = new RegExp('^(?!-)[a-z0-9-s+]{1,32}(?<!-)$');
+export const ARNS_NAME_REGEX = new RegExp('^(?!-)[a-zA-Z0-9-s+]{1,32}(?<!-)$');
+
+export const TIER_DATA: { [x: number]: string[] } = {
+  1: [
+    'Up to 100 Subdomains',
+    'Available via all ArNS-enabled gateways',
+    'Permanently stored on Arweave',
+  ],
+  2: [
+    'Up to 1,000 Subdomains',
+    'Available via all ArNS-enabled gateways',
+    'Permanently stored on Arweave',
+  ],
+  3: [
+    'Up to 10,000 Subdomains',
+    'Available via all ArNS-enabled gateways',
+    'Permanently stored on Arweave',
+  ],
+};
```

Comment:
i think future versions of the contract will have this on them, but fine for now